snippets/standard/datetime/json/custom-reading-with-utf8jsonreader.cs

```diff
@@ -14,6 +14,15 @@ public static void Main(string[] args)
         {
             if (json.TokenType == JsonTokenType.String)
             {
+                try
+                {
+                    DateTimeOffset _ = json.GetDateTimeOffset();
```

Comment:
This seems contrived, and the intent is unclear here: why would anyone ever use a discard in this situation, except to ignore a success and handle a failure?

Reply:
I sort of agree. I would either use a separate sample to show-case the error case (and not in a try/catch), or just call that out in the doc itself rather than adding it to the existing sample for customization.

Reply:
Also, looking at https://docs.microsoft.com/en-us/dotnet/standard/datetime/system-text-json-support, you already have a sample showing what happens when using the `reader.GetDateTime` on an unsupported type.

In that case, this change isn't necessary.